internal/config/config.go

```diff
@@ -34,25 +34,34 @@ type Default struct {
 
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
The struct name `Default` misleads its role.
This should be renamed as `Common` or something like it.

Reply:
fixed